plugins/woocommerce-admin/client/marketplace/components/install-flow/install-new-product-modal.tsx

```diff
@@ -0,0 +1,401 @@
+/**
+ * External dependencies
+ */
+import { ButtonGroup, Button, Modal, Notice } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { dispatch } from '@wordpress/data';
+import { useState, useEffect } from '@wordpress/element';
+import { navigateTo, getNewPath, useQuery } from '@woocommerce/navigation';
+import { recordEvent } from '@woocommerce/tracks';
+import { Status } from '@wordpress/notices';
+
+/**
+ * Internal dependencies
+ */
+import ProductCard from '~/marketplace/components/product-card/product-card';
```

Comment:
I'm going to visit all files and replace the usage of `~` with the relative imports.